networkx/algorithms/find_euler_path.py

```diff
@@ -0,0 +1,223 @@
+import networkx as nx
+
+""" 
+Find an eulerian path
+"""
+#    Copyright (C) 2015 by
+#    Edwardo S. Rivera <edwardo.rivera@upr.edu>
+#    Humberto Ortiz <humberto@hpcf.upr.edu>
+#    BSD license. 
+#    All rights reserved.
+
+
+__author__ = """\n""".join(['Edwardo Rivera <edwardo.rivera@urp.edu>','Humberto Ortiz <humberto@hpcf.upr.edu>'])
+
+__all__ = ['find_eulerian_path'] 
+
+def find_eulerian_path(G):
```

Comment:
I'm not sure what the purpose of the `__author__` variable is; it seems to be an artifact of a time before GitHub. If its purpose is simply to provide contact information for authors of substantial portions of the code, then it makes sense to add your contact information there.